stylelint-polaris/configs/internal.js

```diff
@@ -38,5 +45,34 @@ module.exports = {
         ],
       },
     },
+    [mediaQueriesAllowedList]: [
+      {
+        // Allowed media types and media conditions
+        // https://www.w3.org/TR/mediaqueries-5/#media
+        allowedMedia: [
+          'print',
+          'screen',
+          '-ms-high-contrast',
+          'forced-colors',
+          // TODO: Add utility to @shopify/polaris-tokens to getMediaConditionNames
+          '$p-breakpoints-xs-up',
+          '$p-breakpoints-xs-down',
+          '$p-breakpoints-xs-only',
+          '$p-breakpoints-sm-up',
+          '$p-breakpoints-sm-down',
+          '$p-breakpoints-sm-only',
+          '$p-breakpoints-md-up',
+          '$p-breakpoints-md-down',
+          '$p-breakpoints-md-only',
+          '$p-breakpoints-lg-up',
```

Comment:
Are these suppose to be duplicated or should this be `$p-breakpoints-{up,down,only}` for all?

Reply:
Human error, updated to use RegExp here: https://github.com/Shopify/polaris/pull/6438/commits/6d1f8241e0bd58de5f5fbe4f4c0015a0eb742e61